ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/ShowPopup/ShowPopup.h

```diff
@@ -70,7 +70,7 @@ private slots:
   Esri::ArcGISRuntime::Map* m_map = nullptr;
   Esri::ArcGISRuntime::MapQuickView* m_mapView = nullptr;
   Esri::ArcGISRuntime::PopupManager* m_popupManager = nullptr;
-  QList<Esri::ArcGISRuntime::PopupManager*> m_popupManagers;
+  QList<Esri::ArcGISRuntime::PopupManager*>* m_popupManagers = nullptr;
```

Comment:
A pointer to a QList is unconventional. There is probably a better way to fix the warning you're getting.